infra/cifuzz/actions/entrypoint.py

```diff
@@ -12,46 +12,71 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Builds and runs specific OSS-Fuzz project's fuzzers for CI tools."""
-
+import logging
 import os
-import subprocess
 import sys
 
+# pylint: disable=wrong-import-position
+sys.path.append(
+    os.path.join(os.environ['OSS_FUZZ_HOME'], 'oss-fuzz', 'infra', 'cifuzz'))
+import cifuzz
+
+# TODO: Turn default logging to WARNING when CIFuzz is stable
```

Comment:
Usually the default is info or debug, please change this to info.

Reply:
Discussed offline.